plugins/fabrik_element/jdate/jdate.php

```diff
@@ -2666,6 +2666,21 @@ public function formJavascriptClass(&$srcs, $script = '', &$shim = array())
 			$shim[$key] = $s;
 		}
 
+		/* If the jdate element is loaded on an ajax page, we need to manually include the j! calendar  & javascript files */
+		if (FabrikHelperHTML::inAjaxLoadedPage()) {
+			$min = FabrikHelperHTML::isDebug() ? '.min' : '';
+			$jsloc = 'media/system/js/fields/';
+			$cssloc = 'media/system/css/fields/';
+			echo "<script src='".$jsloc."calendar-locales/date/gregorian/date-helper".$min.".js' type='text/javascript'></script>";
+			echo "<script src='".$jsloc."calendar".$min.".js' type='text/javascript'></script>";
+			echo "<link href='".$cssloc."calendar".$min.".css' rel='stylesheet'>";
+			/* We also need to add them to the shim so they are loaded before the calendar is initialized */
+			$shim[$key]->deps[] = 'media/system/js/fields/calendar.min.js';
```

Comment:
$shim[$key]->deps[] = $jsloc . 'calendar.min.js';
etc.

Reply:
Strange, on line 2672: $jsloc = 'media/system/js/fields/'; so $shim[$key]->deps[] = $jsloc . 'calendar.min.js'; should work